Dispatchfile

```diff
@@ -0,0 +1,28 @@
+#!starlark
+
+gitResource("src-git", url = "$(context.git.url)", revision = "$(context.git.commit)")
+
+def secretVar(name, key):
+    return k8s.corev1.EnvVarSource(secretKeyRef = k8s.corev1.SecretKeySelector(
+        localObjectReference = k8s.corev1.LocalObjectReference(name=name),
+        key = key))
+
+dindTask("dispatch-integration-test",
+         inputs = ["src-git"],
+         steps = [
+             v1.Container(
+                 name="fetch-master",
+                 image = "mesosphere/dispatch-dind:v0.5.2",
+                 workingDir="/workspace/src-git",
+                 args=["git", "fetch", "origin", "master"]),
+             v1.Container(
+                 name = "dispatch-integration-test",
+                 image = "mesosphere/kubeaddons-ci:dispatch",
+                 command = ["make","test"],
+                 workingDir = "/workspace/src-git",
+                 env = [k8s.corev1.EnvVar(
+                     name = "GITHUB_TOKEN",
+                     valueFrom = secretVar("d2iq-dispatch-github-personal-access-token","GITHUB_TOKEN"))])])
```

Comment:
If we're using github-token to pull down dependencies, let's use this opportunity to move away from that pattern. We noted that [here](https://github.com/mesosphere/konvoy/issues/1154) in the konvoy repo

Here's an example of me using ssh key base64 encoded and introduced in an environment variable

Reply:
@faiq  - Ack. better idea.